### ISSUE TITLE
Don't allow to set owner to anyone

### ISSUE THREAD

**Issue by gempir** [SevenTV/ServerGo#1]
If you upload an emote you can set ownership to any users. This shouldn't be possible as it can lead to impersonation. 

Only your channels editors should be allowed to transfer you ownership of emotes.

Otherwise people can upload problematic emotes, transfer ownership to anyone and make it look like another user owns these emotes. 

Example: https://7tv.app/emotes/60aef1a311a994a4ac34caa0

**Comment by AnatoleAM** [comment]
Agreed, will be changing this and most likely requiring the transferee to accept the emote
